addons/account/models/account_tax_carryover_line.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields, api, _
+
+
+class AccountTaxCarryoverLine(models.Model):
+    _name = 'account.tax.carryover.line'
+    _description = 'Tax carryover line'
+
+    name = fields.Char(required=True)
+    amount = fields.Float(required=True, default=0.0)
+    date = fields.Date(required=True, default=fields.Date.context_today)
+    tax_report_line_id = fields.Many2one(
+        string="Tax report line",
+        comodel_name='account.tax.report.line',
+    )
```

Comment:
need something that says it has been carried over or not

Reply:
Well at the moment, it is still working the same way as when using analytic accounts: I'm looking at the balance of the lines before a certain date (the start of the period), and then use that balance to calculate what needs to be done. So a specific line is never really marked as carried over since it'll be canceled by another one.

It seems that you have another idea of how it should work, so I'm open to the suggestion.

Reply:
Not sure either it's necessary. Even a sum from the very first of these carry over lines would work fine, isn't it ?
@william-andre

Reply:
We don't need something to say if it has been carried over, and we don't need `carried_over_date` either.
The amount for a tax line is equal to the amount computed like before, minus the sum of all carryover lines with a `date` prior to the tax report's end date.